jekyll-docs/configuration.md

```diff
@@ -73,7 +73,7 @@ The configuration for  Docker volume plugin require the driver type which can be
   <tbody>
     <tr>
       <td>driver</td>
-      <td>The name of Driver – vsphere for vSphere driver</td>
+      <td>The name of Driver – vsphere for vSphere driver. Possible values are vsphere/vmdk/photon.</td>
```

Comment:
for customer who are still on vmdk and haven't upgraded their scripts ..for backward compatibility reason.